Report.java

```diff
@@ -201,5 +204,61 @@ private void printSpriteCounts(PrintWriter printW, int i)
                 + sprites[j].getSoundBlocksForSprite());
             printW.println();
         }
-    }   
+    }
+
+    /**
+     * Print total variable usage counts.
+     *
+     * @param printW - the printWriter to use
+     * @param i - the submission to get count for
+     */
+    public void printTotalVariableUsage(PrintWriter printW, int i)
```

Comment:
@haietza only thing is, the function declaration for PrintVariableUsageForSprite(blah blah) is too long (more than 80 chars).  Not sure what to do for that.